api/client.yaml

```diff
@@ -1881,11 +2213,85 @@ components:
       required:
         - addressID
         - type
+        - ownerType
         - address1
         - city
         - state
         - postalCode
         - country
+    CreateCustomerRepresentative:
+      properties:
+        firstName:
+          type: string
+          description: Given Name or First Name
+          example: Mary
+        lastName:
+          type: string
+          description: Surname or Last Name
+          example: Smith
+        jobTitle:
+          type: string
+          description: Job title of this representative
+          example: Chief Executive Officer
+        birthDate:
+          type: string
+          description: Legal date of birth
+          example: '2016-08-29'
+        SSN:
```

Comment:
Is this really a required field? There's a need to store the SSN of CEO's who use Customers?

Reply:
Where does it say it's required?  It should be an optional field, and it's not in the list of required fields for the API request

Reply:
Oh right. I think it should be `nullable: true` then?

Reply:
It looks like in the model it does have the `omitempty` part, so it is already set to `nullable: true`